notes

```diff
@@ -0,0 +1,6 @@
+Meeting summary:
```

Comment:
Do we need this file?

Reply:
Oops, nope! That represents some pending action items on my part re: devops. Thanks for spotting :)